src/lambdas/users-delete.js

```diff
@@ -0,0 +1,57 @@
+import ExceptionHandler from '@/exceptions/handler';
+import ResourceNotFoundException from '@/exceptions/resource-not-found';
+import User from '@/models/user';
+import {DataMapper} from '@aws/dynamodb-data-mapper';
+
+const DyanamoDB = require('aws-sdk/clients/dynamodb');
+const Lambda = require('aws-sdk/clients/lambda');
+
+/**
+ * @api {delete} /users/:id Delete a User
+ * @apiGroup Users
+ *
+ * @apiExample {curl} Example
+ * curl -X DELETE -H 'x-api-key: API_KEY_HERE' {{API_URL}}/users/bde526c8-9b59-45dc-9551-f8996d091fdc
+ *
+ * @apiSampleRequest off
+ *
+ * @apiUse Headers
+ *
+ * @apiParam {String} id User's unique identifier.
+ *
+ * @apiSuccessExample Success-Response
+ * HTTP/1.1 204 No Content
+ *
+ * @apiUse ResourceNotFoundException
+ * @apiUse Exception
+ */
+export async function handler(event, context, callback) {
+	console.log(event);
```

Comment:
Since your logs are being sent to AWS Cloudwatch, how would you go about implementing a pattern which is consistent across logging statements? Is there a transporter with a pattern you would suggest?

Reply:
A recommended approach is to introduce something like [winston](https://github.com/winstonjs/winston) for logging. Then defining CloudWatch patterns for errors or general logging would be more straight forward and consistent.